inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseProfileDeserializationTest.java

```diff
@@ -55,17 +59,18 @@ public void checkThatDeserializationWorks() throws JsonParseException, JsonMappi
         String descriptionIri = "http://www.w3.org/2000/01/rdf-schema#comment";
         String propertyLabelIri = "http://www.w3.org/2000/01/rdf-schema#label";
         String propertyDescriptionIri = "http://www.w3.org/2000/01/rdf-schema#comment";
-
+        List<String> rootConcepts =  new ArrayList<>(Arrays.asList(OWL.NOTHING.stringValue()));
```

Comment:
`asList(OWL.NOTHING.stringValue())` (with a static import of `Arrays.asList`) is sufficient unless you need the list to be mutable (which I do assume is not the case).